src/main/webapp/app/exam/exam-scores/exam-scores.component.ts

```diff
@@ -103,14 +117,80 @@ export class ExamScoresComponent implements OnInit, OnDestroy {
         this.changeDetector.detectChanges();
     }
 
+    private createChart() {
+        const labels = [];
+        let i;
+        for (i = 0; i < this.histogramData.length; i++) {
+            if (i === this.histogramData.length - 1) {
+                labels[i] = `[${i * this.binWidth},${(i + 1) * this.binWidth}]`;
+            } else {
+                labels[i] = `[${i * this.binWidth},${(i + 1) * this.binWidth})`;
+            }
```

Comment:
Took me some time to get the difference between the two assignments :-D Maybe the following is easier to parse:

```suggestion
            labels[i] = `[${i * this.binWidth},${(i + 1) * this.binWidth}`;
            labels[i] += (i === this.histogramData.length - 1) ? ']' : ')';
```